fe2-android/app/src/main/java/com/github/dedis/popstellar/utility/ActivityUtils.kt

```diff
@@ -0,0 +1,287 @@
+package com.github.dedis.popstellar.utility
+
+import android.app.Activity
+import android.app.Application
+import android.content.Context
+import android.content.res.Configuration
+import android.graphics.Color
+import android.os.Bundle
+import android.view.View
+import android.widget.ImageView
+import android.widget.TextView
+import androidx.activity.OnBackPressedCallback
+import androidx.fragment.app.Fragment
+import androidx.fragment.app.FragmentManager
+import androidx.lifecycle.Lifecycle
+import com.github.dedis.popstellar.model.objects.Wallet
+import com.github.dedis.popstellar.model.objects.security.Base64URLData
+import com.github.dedis.popstellar.repository.database.subscriptions.SubscriptionsDao
+import com.github.dedis.popstellar.repository.database.subscriptions.SubscriptionsEntity
+import com.github.dedis.popstellar.repository.database.wallet.WalletDao
+import com.github.dedis.popstellar.repository.database.wallet.WalletEntity
+import com.github.dedis.popstellar.repository.remote.GlobalNetworkManager
+import io.github.novacrypto.bip39.MnemonicGenerator
+import io.github.novacrypto.bip39.wordlists.English
+import io.reactivex.android.schedulers.AndroidSchedulers
+import io.reactivex.disposables.Disposable
+import io.reactivex.schedulers.Schedulers
+import java.security.GeneralSecurityException
+import java.security.MessageDigest
+import java.security.NoSuchAlgorithmException
+import java.util.Arrays
+import java.util.Collections
+import java.util.function.Consumer
+import java.util.function.Supplier
+import kotlin.math.abs
+import timber.log.Timber
+
+object ActivityUtils {
```

Comment:
Outside the scope of this PR:
this object seems to be getting quite bit and with a bunch of unrelated things, i.e. `buildBackButtonCallback` doesn't exactly seem to fit.

Could we break this into smaller, more narrowly-focused objects ?